<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultDialog</class>
 <widget class="QDialog" name="ResultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>不好意思,您输了.下次走运!</string>
   </property>
  </widget>
  <widget class="QLabel" name="playTimeLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>60</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="bestTimeLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="countPlayLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>130</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="countWinLabel">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>170</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="gameInfoLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>145</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetGameButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>145</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>重新开始这个游戏(&amp;R)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="againButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>200</y>
     <width>145</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>在玩一局(&amp;P)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
